js/dom.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable vars-on-top */
+/* eslint-disable no-var */
+/* eslint-disable no-undef */
+/* eslint-disable no-use-before-define */
```

Comment:
Why did you disable so many rules?

Reply:
Files are updated now we just disable  the rules: 

/* eslint-disable no-undef */
/* eslint-disable no-unused-vars */

To prevent false positives since we are using a js script for dom and another for logic.